src/validation/val/val.coffee

```diff
@@ -0,0 +1,40 @@
+validation = (validation) ->
+  auth = validation.auth
+  validation.validation =
+    add_team: (team) ->
+      return true
+    remove_team: (team) ->
```

Comment:
This is just validation. So, whether the action is valid, not whether the user is authorized to perform the action. There is no validation necessary, right now, for add_team. Therefore, it always validates to true. Authorization is an entirely different matter. Does that answer the question?